### PR TITLE
Bug/eric/fix signin returned data

### PR DESCRIPTION
**Pull request by EngrEric**

### CODE REVIEW

**Comment by Simpcyclassy** [comment]
Why did you create a new pull request

**Comment by EngrEric** [comment]
i created the branch to fix the json data returned from signin endpoint. I included the user info as more payload from the endpoint.

**Comment by EngrEric** [comment]
> Why did you create a new pull request? Everything looks same, except helpers.js. But if this is different I suppose there ought to be patch increment here too.

i created the branch to fix the json data returned from signin endpoint. I included the user info as more payload from the endpoint.

File: routes/signInAndOut/helpers.js [hunk 1]
```diff
@@ -71,9 +71,47 @@ export default {
...
+                    country,
+                    createdAt,
+                    email,
+                    firstName,
+                    lastName,
+                    phoneNumber,
+                    state,
+                    updatedAt,
+                    username,
+                }); // will redo this, because i only want to take out password from the user object
```
Comment (enyeInc) [inline routes/signInAndOut/helpers.js]: can you use lodash.pick to select the keys that you want?

**Comment by enyeInc** [comment]
@EngrEric increment the version

**Pull request closed without merging**